src/panel/applets/trash/trash_applet.c

```diff
@@ -17,6 +17,7 @@
 #include "trash_notify.h"
 #include "trash_popover.h"
 #include "trash_settings.h"
+#include <budgie-enums.h>
```

Comment:
This doesn't feel like the correct header. It should be `popover.h`, no?

Reply:
Fair enough - compilation works either way.  So pushed that instead.

Still doesnt though fix the underlying issue of src/windowing not being compiled correctly into the icon-tasklist :(